src/Tokenizer.ts

```diff
@@ -654,8 +654,7 @@ export class _Tokenizer {
 
         // Remove extra characters. *a*** -> *a*
         rLength = Math.min(rLength, rLength + delimTotal + midDelimTotal);
-
-        const raw = [...src].slice(0, lLength + match.index + rLength + 1).join('');
+        const raw = src.slice(0, lLength + match.index + rLength + [...match[0]][0].length);
```

Comment:
Is the spread necessary here?

Reply:
Yes, that is so unicode character length is calculated correctly.

Reply:
It might be good to add a comment here since it’s not obvious